rtmp/src/main/java/com/github/faucamp/simplertmp/io/RtmpConnection.java

```diff
@@ -362,6 +366,24 @@ private void onMetaData() {
     ecmaArray.setProperty("audiodatarate", 0);
     ecmaArray.setProperty("stereo", true);
     ecmaArray.setProperty("filesize", 0);
+
+    if (addlMetaData instanceof JSONObject) {
```

Comment:
Let me know if I should change this Object/JSONObject process.

Reply:
I think JSONObject works fine!